package.json

```
@@ -655,17 +655,17 @@
                 },
                 {
                     "command": "appService.Refresh",
-                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected$/",
+                    "when": "view == azureAppService && viewItem =~ /^deployments(C|Unc)onnected(|NoPortal)$/",
```

Curious if Nathan agrees, but the "NoPortal" thing in the context value feels a bit overly complex to me. I think I'd rather let it be shown on trial apps and simply throw an error in that case

---

Another alternative: Trial apps don't have the deployments node

---

I think I agree with Eric.  Does redeploy work for trial apps?  I think the logs could be useful though.

---

Redeploy works for trial apps.

---

Yeah I feel like it's worth it for the deployment logs

---

Let's just throw an error then